### PR TITLE
Common: changed civetweb line in rgw section(conf)

### PR DESCRIPTION
**Pull request by SirishaGuduru**
Resolves issue: Multiple RGW Ceph.conf Issue #1258

In multi-RGW setup, in ceph.conf the RGW sections
contain identical bind IP in civetweb line. So this
modification fixes that issue and puts the right IP
for each RGW.

Signed-off-by: SirishaGuduru SGuduru@walmartlabs.com

### CODE REVIEW

**Comment by guits** [comment]
jenkins test luminous-ansible2.2-centos7_cluster

**Comment by guits** [comment]
jenkins test jewel-ansible2.2-centos7_cluster

**Comment by guits** [comment]
jenkins test jewel-ansible2.2-centos7_cluster

**Comment by leseb** [comment]
jenkins test this please

File: roles/ceph-common/templates/ceph.conf.j2 [hunk 1]
```diff
@@ -135,7 +135,11 @@ keyring = /var/lib/ceph/radosgw/{{ cluster }}-rgw.{{ hostvars[host]['ansible_hos
 rgw socket path = /tmp/radosgw-{{ hostvars[host]['ansible_hostname'] }}.sock
 log file = /var/log/ceph/{{ cluster }}-rgw-{{ hostvars[host]['ansible_hostname'] }}.log
 rgw data = /var/lib/ceph/radosgw/{{ cluster }}-rgw.{{ hostvars[host]['ansible_hostname'] }}
-rgw frontends = civetweb {{ radosgw_civetweb_options }}
+{% if ip_version == 'ipv6'  %}
+rgw frontends = civetweb port=[{{ hostvars[host]['ansible_'+ monitor_interface]['ipv6'][0]['address'] }}]:{{ radosgw_civetweb_port }} {{ radosgw_civetweb_options }}
```
Comment (guits) [inline roles/ceph-common/templates/ceph.conf.j2]: @SirishaGuduru please, take a look at [this](https://github.com/SirishaGuduru/ceph-ansible/blob/341af20e6820a924df11cb983aaa674a07954606/roles/ceph-common/templates/ceph.conf.j2#L67-L73) to make it the same way

File: roles/ceph-common/templates/ceph.conf.j2 [hunk 2]
```diff
@@ -135,7 +135,11 @@ keyring = /var/lib/ceph/radosgw/{{ cluster }}-rgw.{{ hostvars[host]['ansible_hos
...
+{% else %}
+rgw frontends = civetweb port={{ hostvars[host]['ansible_'+ monitor_interface]['ipv4']['address'] }}:{{ radosgw_civetweb_port }} {{ radosgw_civetweb_options }}
```
Comment (guits) [inline roles/ceph-common/templates/ceph.conf.j2]: ditto

**Comment by SirishaGuduru** [comment]
@guits , please review the changes.

**Comment by leseb** [comment]
@SirishaGuduru please, rebase.

**Pull request closed without merging**